### ISSUE TITLE
[Feature] Video Upscaler

### ISSUE THREAD

**Issue by martineliascz** [chaosaiart/Chaosaiart-Nodes#5]
Hi, your nodes & workflows look good!

Can they be used to create basic video upscaler workflow: drag-n-drop video -> convert into upscaled frames -> stitch all frames back into one video and add audio back from the drag-n-drop video?

If so, any ideas how to achieve this?

Thanks!

**Comment by chaosaiart** [comment]
-> _convert into upscaled frames -> stitch all frames back into one video_
The functions are there. 
However, you would need three individual steps that could be built together with code. 
1. Decompose 
2. Run through all frames 
3. Reassemble. 
For steps 1 and 3, I have Python scripts that I modified so they fit into ComfyUI and can be used by everyone. 
Step 2, using the upscaling of video input without losing frame-to-frame similarity, will be the challenge. The functions are there.
  
-> _add audio back from the drag-n-drop video?_
Currently, no audio track is considered in my custom nodes. 
I have already extracted audio and later rebuilt it in another project, my AI Anime Dubs. 
So I have code somewhere that could be used for this, but I wouldn't know where to meaningfully integrate such a function into my custom node.
